apps/prairielearn/src/server.ts

```diff
@@ -2191,7 +2195,7 @@ if (esMain(import.meta) && config.startServer) {
       directories: [path.join(import.meta.dirname, 'batched-migrations')],
     });
 
-    runner.on('error', (err) => {
+    runner?.on('error', (err) => {
```

Comment:
When would `runner` ever not be initialized here?

Reply:
Re-initialization of HMR.